assets/scss/_registry.scss

```diff
@@ -39,3 +39,13 @@
 .registry-entry {
   @extend .shadow;
 }
+
+// Scroll-to-top button styling
+
+@media screen and (max-width: 768px) {
+  .scroll-to-top-btn {
+    bottom: 20px;
+    right: 20px;
+    padding: 10px;
+  }
+}
```

Comment:
`.scroll-to-top-btn` doesn't appear to be used (and if it were, why define it inside a media query?).

Reply:
the styling was needed for the scroll-to-top-btn mobile view

Reply:
On reviewing myself you indeed were right, it actually was not needed, more like dead-code. Its been removed and the styling adjustment is fully applied. Appreicate you pointing that out mentor! @chalin